.github/scripts/sync-up-dotnet-ver.js

```diff
@@ -0,0 +1,150 @@
+const fse = require("fs-extra");
+const path = require("path");
+const semver = require("semver");
+const targetPkgName = process.argv[2];
+const targetPath = path.join(__dirname, "../../packages", targetPkgName);
+const xml2js = require(path.join(targetPath, "node_modules/xml2js"));
```

Comment:
Can the npm pacakge be required using 
```
var parseString = require('xml2js').parseString;
```

Reply:
no
cause xml2js is a lib file, this script locations is not under function-extension or simpleauth folder, so the env $PATH can not concat the relative path to xml2js.